xmake/modules/package/manager/find_package.lua

```diff
@@ -65,15 +64,16 @@ function _find_package_with_builtin_rule(package_name, opt)
     end
 
     -- find package from the given package manager
-    local result = nil
+    local result, manager_name = nil, nil
     for _, manager_name in ipairs(managers) do
         dprint("finding %s from %s ..", package_name, manager_name)
         result = import("package.manager." .. manager_name .. ".find_package", {anonymous = true})(package_name, opt)
         if result then
             break
         end
+        manager_name = nil
```

Comment:
```lua
    local result = nil
    local manager_found = nil
    for _, manager_name in ipairs(managers) do
        dprint("finding %s from %s ..", package_name, manager_name)
        result = import("package.manager." .. manager_name .. ".find_package", {anonymous = true})(package_name, opt)
        if result then
            manager_found = manager_name
            break
        end
    end
    return result, manager_found
```

Reply:
Sorry about the variable shadowing on the for loop variable. Changed to your working/cleaner version!